modules/Microsoft.SignalRService/webPubSub/.test/parameters.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
If private endpoints are available, should we add them to the main test?

Reply:
Definitiely. On it

Reply:
done